whats_on/app/views/pages/installation.njk

```diff
@@ -0,0 +1,69 @@
+{% extends 'layout/default.njk' %}
+{% block pageMeta %}
+  {% set metaContent = {
+    type: 'website',
+    title: installation.title,
+    description: installation.promo.caption,
+    url: pageConfig.canonicalUri
+  } %}
+  {% component 'open-graph', { pageConfig: pageConfig, metaContent: metaContent } %}
+  {% component 'twitter-card', { pageConfig: pageConfig, metaContent: metaContent } %}
+  <meta name="description" content="{{ installation.description | prismicAsText | safe | striptags }}" />
+{% endblock %}
+
+{% block body %}
+  {# {% set jsonLdExhibition = exhibition | objectAssign({safeDescription: exhibition.description | safe | striptags}) %} #}
+
+  <script type="application/ld+json">
+    {# {{ jsonLdExhibition | jsonLd('exhibitionLd') | safe }} #}
+  </script>
+  <div class="row relative">
+    <div class="flush-container-left absolute {{ {s:2} | spacingClasses({margin: ['top', 'bottom']}) }}">
+      {% componentJsx 'Tags', { tags: tags } %}
+    </div>
+
+    <div class="exhibition-hero">
+      {% componentJsx 'Picture', {
+        images: featredImageList,
+        extraClasses: 'exhibition-hero__picture'
+      }%}
+
+      <div class="exhibition-hero__copy {{ {l:10} | spacingClasses({margin: ['bottom']}) }}">
+        <div class="is-hidden-l is-hidden-xl">
+          {% component 'wobbly-edge', {background: 'white'} %}
+        </div>
+
+        <div class="container">
+          <div class="grid">
+            <div class="{{ {s:12, m:10, shiftM:1, l:12, xl:12} | gridClasses }}">
+              <div class="bg-white inline-block exhibition-hero__box {{ {l:4} | spacingClasses({'padding': ['right', 'left']}) }} {{ {s:4} | spacingClasses({'padding': ['top', 'bottom']}) }} rounded-diagonal">
+                <div class="flex flex--v-center flex--h-space-between">
+                  <span class="{{ {s:'HNM4', m:'HNM3'} | fontClasses }} {{ {s: 6} | spacingClasses({margin: ['right']}) }}">Free admission</span>
+                  {# <span class="flex flex--v-center {{ {s:'HNM4'} | fontClasses }}">
```

Comment:
Just checking this commented stuff is supposed to be checked in ?

Reply:
Yea, it's removed later on.